### PR TITLE
Release v3.0.0

### PR DESCRIPTION
**Pull request by lukpueh**
* Bump version
* Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v3.0.0
+
+This release, most notably, marks stable securesystemslib v1.0.0 as minimum
+requirement. The update requires the previously announced removal of deprecated
```
Comment (adityasaky) [inline CHANGELOG.md]:
```suggestion
requirement. The update incorporates the previously announced removal of deprecated
```
 
perhaps? "requires" sounds funny

**Review by adityasaky — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (lukpueh) [reply CHANGELOG.md]: Though wait, I meant that when we update to securesystemslib v1.0.0, we must remove the deprecated args. So the removal is not really incorporated by the dependency update, but necessitated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+CLI/API arguments (see below).
```
Comment (lukpueh) [inline CHANGELOG.md]: What about: 

```suggestion
requirement, and, as a consequence, removes the recently deprecated CLI/API arguments (see below).
```

Comment (adityasaky) [reply CHANGELOG.md]: looks good to me.

**Pull request merged**